templates/base/packages/nextjs/services/web3/wagmiConfig.tsx.template.mjs

```diff
@@ -0,0 +1,35 @@
+import { deepMerge, withDefaults, stringify } from "../../../../../utils.js";
+
+const defaultWagmiConfig = {
+  chains: '$$enabledChains$$',
+  connectors: '$$wagmiConnectors$$',
+  ssr: true,
+  client: `$$({ chain }) => { let rpcFallbacks = [http()]; const rpcOverrideUrl = (scaffoldConfig.rpcOverrides as ScaffoldConfig["rpcOverrides"])?.[chain.id]; if (rpcOverrideUrl) { rpcFallbacks = [http(rpcOverrideUrl), http()]; } else { const alchemyHttpUrl = getAlchemyHttpUrl(chain.id); if (alchemyHttpUrl) { const isUsingDefaultKey = scaffoldConfig.alchemyApiKey === DEFAULT_ALCHEMY_API_KEY; rpcFallbacks = isUsingDefaultKey ? [http(), http(alchemyHttpUrl)] : [http(alchemyHttpUrl), http()]; } } return createClient({ chain, transport: fallback(rpcFallbacks), ...(chain.id !== (hardhat as Chain).id ? { pollingInterval: scaffoldConfig.pollingInterval, } : {}), }); }$$`,
```

Comment:
We neeed to inline the whole function in one line to make it work, and the people who are passing it in `.args.mjs` file need to do the same: 

https://github.com/scaffold-eth/create-eth-extensions/blob/2ef372d4cd26788251cfbabf88f5f70da4a6cfe7/extension/packages/nextjs/services/web3/wagmiConfig.tsx.args.mjs#L5